package.json

```diff
@@ -1,13 +1,34 @@
 {
   "name": "fluent-javascript",
-  "version": "0.0.1",
   "private": true,
+  "workspaces": [
+    "./fluent-dedent",
+    "./fluent-bundle",
+    "./fluent-sequence",
+    "./fluent-dom",
+    "./fluent-langneg",
+    "./fluent-react",
+    "./fluent-syntax",
+    "./fluent-gecko"
+  ],
+  "scripts": {
+    "predist": "npm run clean",
+    "dist": "npm run build --workspaces",
+    "postdist": "npm run lint && npm run test && npm run docs --workspaces",
+    "clean": "git clean -fdxe node_modules fluent-*/{coverage,dist,esm,index.js} html",
+    "deploy-html": "gh-pages -d html",
```

Comment:
Issue: This one failed too:

```
➤ npm run deploy-html

> deploy-html
> gh-pages -d html

ENOENT: no such file or directory, stat '/Users/greg/dev/fluent.js/html'
```

I think we need a `mkdir html` here.

Reply:
The `html` directory is generated by the packages' `docs` scripts, which are run as a part of `postdist`. Running this script should then update your local `gh-pages` branch and push that to GitHub.

So it's probably a good thing that this failed for you, as on success it would've updated the contents that we're serving under https://projectfluent.org/fluent.js/.